src/lib/template-result.ts

```diff
@@ -43,33 +43,34 @@ export class TemplateResult {
    * Returns a string of HTML used to create a `<template>` element.
    */
   getHTML(): string {
-    const endIndex = this.strings.length - 1;
+    const l = this.strings.length - 1;
     let html = '';
-    for (let i = 0; i < endIndex; i++) {
+    let isTextBinding = true;
```

Comment:
We could change this to look for comment bindings instead?

```js
let isCommentBinding = false;
for (let i = 0; i < l; i++) {
  const s = this.strings[i];
  const open = s.lastIndexOf('<!--');
  isCommentBinding = (open > -1 || isCommentBinding) &&
    s.indexOf('-->', open + 1) === -1;

  if (isCommentBinding) {
    html += s + marker;
    continue;
  }
  const match = lastAttributeNameRegex.exec(s);
  if (match) {
    html += s.substr(0, match.index) + match[1] + match[2] +
      boundAttributeSuffix + match[3] + marker;
  } else {
    html += s + nodeMarker;
  }
}
```

Reply:
Thanks! Added this with a couple of changes